server/src/main/resources/openapi/schemas/sharedIndexTitle.json

```diff
@@ -31,6 +27,6 @@
   },
   "additionalProperties": false,
   "required": [
-    "globalId", "localId", "matchKeys", "payload"
+    "globalId", "localId", "sourceId"
   ]
 }
```

Comment:
Consider renaming the schema to "globalRecord" as the term "sharedIndexTitle" is not aligned with the terms used in the code.

Reply:
Right

Reply:
Renamed